bin/git-ignore-io

```diff
@@ -0,0 +1,124 @@
+#/bin/sh
+
+gitignore_io_url="https://www.gitignore.io/api/"
+
+default_path="$HOME/.gi_list"
+gi_list=`cat $default_path | tr "," "\n"`
```

Comment:
Another way to get `gi_list`:
`gi_list=$(sed 's/,/\n/g' "$default_path")`

Or:
`gi_list=$(sed 's/,/ /g' "$default_path")`. It seems no item contains whitespace.